mesh/nurbs.hpp

```diff
@@ -67,6 +67,15 @@ class KnotVector
    void CalcD2Shape(Vector &grad2, int i, double xi) const
    { CalcDnShape(grad2, 2, i, xi); }
 
+   /** Gives the locations of the maxima of the knotvector in reference space. The
+      function gives the knotspan @a ks, the coordinate in the knotspan @a xi
+      and the coordinate of the maximum in parameter space @a u */
+   void FindMaxima(Array<int> &ks, Vector &xi, Vector &u);
+   /** Global curve interpolation through the points @a x. @a x is an array with the
+      length of the spatial dimension containing vectors with spatial coordinates. The
+      controlpoints of the interpolated curve are given in @a x in the same form.*/
+   void FindInterpolant(Array<Vector*> &x);
```

Comment:
I would prefer `std::vector<Vector>` over `Array<Vector*>`, but the latter is consistent with the existing code, so we can keep it as is for now. Maybe it is too late to change the API.

Reply:
Good suggestion! I think I do not have any useful arguments to add to this comment.